Sources/TerminalAPIKit/Models/Session.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Session.swift
+//  TerminalAPIKit
+//
+//  Created by Andrew Gates on 8/8/23.
+//
+
+import Foundation
+
+public final class Session: Codable {
+    
+    /// The unique reference of the session.
+    public let id: Int
+    
+    /// The type of the transaction. Use ``SessionType/Begin`` to start a session, and ``SessionType/End`` to end the session.
+    public let type: SessionType
+    
+    /// Initializes the Session.
+    ///
+    /// - Parameter id: Undocumented.
```

Comment:
can we write also document the parameters here? applies in multiple places in the PR

Reply:
Latest commit should handle these